### PR TITLE
Explicitly set tmp dir for fasterq-dump

### PR DESCRIPTION
**Pull request by mvdbeek**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/sra-tools/fasterq_dump.xml [hunk 1]
```diff
@@ -13,7 +14,9 @@
     @CONFIGURE_RETRY@
     @SET_ACCESSIONS@
     while [ \$SRA_PREFETCH_ATTEMPT -le \$SRA_PREFETCH_RETRIES ] ; do
-        fasterq-dump "\$acc" -e \${GALAXY_SLOTS:-1}
+        fasterq-dump "\$acc"
+        -e \${GALAXY_SLOTS:-1}
+        -t \${TMPDIR}
```
Comment (bernt-matthias) [inline tools/sra-tools/fasterq_dump.xml:19]: Maybe 

```suggestion
        -t \${TMPDIR:./}
```
or 

```suggestion
        -t \${TMPDIR:\$_GALAXY_JOB_TMP_DIR}
```

Comment (mvdbeek) [reply tools/sra-tools/fasterq_dump.xml:19]: We're always setting it, so the fallback seems confusing ?

File: tools/sra-tools/fasterq_dump.xml [hunk 2]
```diff
@@ -2,6 +2,7 @@
     <description>format from NCBI SRA</description>
     <macros>
         <import>macros.xml</import>
+        <token name="VERSION_SUFFIX">1</token>
```
Comment (bernt-matthias) [inline tools/sra-tools/fasterq_dump.xml:5]: Does this "overwrite" the token in macros.xml? 

Is this a good idea?

Comment (mvdbeek) [reply tools/sra-tools/fasterq_dump.xml:5]: yes. why should we bump every tool if only one has changed ?

**Comment by mvdbeek** [comment]
```
Downloading accession: SRR11953971...
Failed to call external services.
Prefetch attempt 1 of 3 exited with code 1
Failed to call external services.
Prefetch attempt 2 of 3 exited with code 1
Failed to call external services.
Prefetch attempt 3 of 3 exited with code 1
```
that's fun ...

File: tools/sra-tools/fasterq_dump.xml (hunk 2 above)
Comment (bernt-matthias) [reply tools/sra-tools/fasterq_dump.xml:5]: I understand the argument, but IMO this is easily missed when updating the macros.

Comment (mvdbeek) [reply tools/sra-tools/fasterq_dump.xml:5]: Isn't that handled by the bot ? What's the process to just bump one tool ?

Comment (bernt-matthias) [reply tools/sra-tools/fasterq_dump.xml:5]: Sometimes people bump manually :)

Comment (mvdbeek) [reply tools/sra-tools/fasterq_dump.xml:5]: ok, so now what ? I can drop `@VERSION_SUFFIX@ ` and put a literal 1 if that's any better

**Comment by bernt-matthias** [comment]
Seems that this did not work. At least the [toolshed](https://toolshed.g2.bx.psu.edu/view/iuc/sra_tools/1ec814014652) still reports this as `galaxy0`. Also the complete repo including all three tools has been updated, or? Was a bit surprised that fasterq-dump can't be found at all as a single tool.

**Pull request merged**